fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraLdp.java

```diff
@@ -189,11 +200,33 @@ public Response head() {
 
             // we set the content-type explicitly to avoid content-negotiation from getting in the way
             builder.type(mediaType.toString());
+        } else {
+            final MediaType mediaType = RDFMediaType.TURTLE_TYPE;
+            final RdfStream stream = getResourceAsStream();
```

Comment:
@acoburn :  okay.  That all seems reasonable.  In the future I'll double check on IRC if I'm unsure about how to approach an issue before diving in.   That said,  I think the other refactoring s are worthwhile.   I'll pull out the change to the HEAD behavior and reissue another pull request for the other changes.